### PR TITLE
Added 1 Javascript Question

### PR DESCRIPTION
**Pull request by P766-spec**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by P766-spec** [comment]
Added 1 javascript Question related to in operator

**Comment by P766-spec** [comment]
Added 1 sql question related to super key

**Comment by P766-spec** [comment]
Hello @jdwilkin4 please checkout the added questions on js and sql!  Would love to contribute on more, Thank You.

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "What keyword is used to check whether a given property is valid or not?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:4]:
```suggestion
      "In JavaScript, what keyword is used to check whether a given property in an object is valid?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    Answer: "in",
+    Distractor1: "is in",
+    Distractor2: "exists",
+    Distractor3: "lies",
+    Explanation:
+      "The in keyword is used to check whether the given property is valid or not in Javascript.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:10]:
```suggestion
      "The in keyword is used to check whether the given property in an object is valid or not.",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+    Question: "Which of the following allows you to uniquely identify a tuple?",
+    Answer: "Super Key",
+    Distractor1: "Schema",
+    Distractor2: "Attribute",
+    Distractor3: "Domain",
+    Explanation:
+      "Super Key is used to uniquely identify a tuple in a relation.",
+    Link: "https://en.wikipedia.org/wiki/Superkey",
+  },
+
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:12]: please remove this extra space here

File: src/data/sql-quiz.ts [hunk 4]
```diff
@@ -1,11 +1,24 @@
 const sqlQuiz = [
   {
-    Question: "Which command is used in a 'WHERE' or 'HAVING' clause to limit the selected rows to the items when a column has a certain pattern of characters contained in it?",
+    Question: "Which of the following allows you to uniquely identify a tuple?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:3]:
```suggestion
    Question: "In relational databases, which of the following allows you to uniquely identify a tuple?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by P766-spec** [comment]
Okkk!!!

**Comment by P766-spec** [comment]
@jdwilkin4 I have done the requested changes, please review it and merge the PR.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**